src/epochs.lua

```diff
@@ -779,9 +775,10 @@ function convertPrescribedEpochToDistributedEpoch(epoch, currentTimestamp, distr
 	}
 end
 
+--- TODO: consider removing this, epochs should manage their own lifecycle and not require prune
```

Comment:
How will the new pattern change the way event data is reported around pruning?

Reply:
the distribution/creation should provide the event data on the distributed/new epoch

Reply:
removed pruning of epochs in https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f

tick should manage epochs in state